src/test/inspection/renameEdits.ts

```diff
@@ -218,6 +218,39 @@ describe(`Inspection - RenameEdits - Identifiers`, () => {
             assertExpectedRecordTextEdits(textEdits, nextStr);
         });
 
+        it(`Rename one identifier in its paired assignment expression creator`, async () => {
+            const rawText: string = [
```

Comment:
nit: could you make this one big, formatted string so it's easier to read? Same goes with the other tests E.g.

![image](https://user-images.githubusercontent.com/17787492/166568295-30bea878-02d2-4edf-b13c-ee7f435c902e.png)

As-is I don't at a glance can't tell what layer is which record. It could be my vision that's the problem, but I'm guessing there's other people with vision issues out there :)

Reply:
understood~